entity-types/infra-mongodb_collection/definition.yml

```diff
@@ -6,7 +6,6 @@ synthesis:
     - compositeIdentifier:
         separator: ":"
         attributes:
-          - targetName
           - mongodb_cluster_name
           - database
           - collection
```

Comment:
Is unrelated but, perhaps adding `database` and `collection` to the entity tags too? I'm thinking that they could be good to filter them.

Reply:
I'll add those